### ISSUE TITLE
PDF-file doesn't open

### ISSUE THREAD

**Issue by ghost** [manuels/texlive.js#20]
Thanks for creating texlive.js, it's great!

There is one problem: It doesn't work on Chromium under Ubuntu 14.04. The compiling seems to work, but the PDF is not opened. Instead, "Cannot GET /false" is displayed".

**Comment by fzimmermann89** [comment]
Is this issue still there?

**Comment by igauravsehrawat** [comment]
Yes, on opening console. I get `Not allowed to navigate top frame to data URL: `

**Comment by thealexinator2904** [comment]
I found out, that refreshing the opened page helps on chrome. However I don't think thats satisfying.
